recipes/pyhalcyon/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pyhalcyon" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e84740fd2f319ae2cdc017c0ba5eba284bc4aa13df95debea9a35f04e4cc7aa7
+
+
+build:
+  number: 0
+  entry_points:
+    - halcyon = halcyon.console:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python ==3.7.4
```

Comment:
you should not pin python that strict, does this tool work with python3 in general?